src/scripts/main.js

```diff
@@ -5,4 +5,30 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const current = document.querySelector('tr');
+
+people.forEach((person) => {
+  person.age = person.died - person.born;
+  person.century = Math.ceil(person.died / 100);
+
+  if (person.sex === 'f') {
+    person.sex = 'female';
+  } else {
```

Comment:
redundant to use if else. better to use ternary operator
Female and Male should be with Capital first letter